src/main/java/org/truffleruby/parser/Translator.java

```diff
@@ -60,6 +60,7 @@
         m.put("$INPUT_RECORD_SEPARATOR", "$/");
         m.put("$>", "$stdout");
         m.put("$PROGRAM_NAME", "$0");
+        m.put("$CHILD_STATUS", "$?");
```

Comment:
@eregon `English.rb` aliasing doesn't appear to have been fully implemented yet. The current solution kind of looks like a workaround. It'd be nice if aliasing was more implemented but I'm happy if this workaround lets `mysql2` install for now.